projects/next-dropzone/src/lib/services/next-drag-and-drop.service.ts

```diff
@@ -0,0 +1,15 @@
+import {Injectable, HostBinding} from '@angular/core';
+
+@Injectable()
+export class NextDragAndDropService {
+  public removeBorder: boolean = false;
+
+  @HostBinding('style.background') public background;
+  @HostBinding('style.border') public border;
+  @HostBinding('style.border-radius') public borderRadius;
+  @HostBinding('style.border-color') public borderColor;
+
+  public onDrop(): void {
+    this.removeBorder = true;
+  }
```

Comment:
Okay, I really don't see any reasons to add this method into the service but maybe it's because of my not-enough-experience in Angular :)

Reply:
Because service is singleton, add it helps catch drop on other elements

Reply:
oh, thx I'll try to understand it (I almost did it).